dmutils/content_loader.py

```diff
@@ -63,14 +63,6 @@ def get_section_filtered_by(self, section_id, service_data):
         else:
             return None
 
-    def _yaml_file_exists(self, yaml_file):
-        return os.path.isfile(yaml_file)
-
-    def _read_yaml_file(self, yaml_file):
-        with open(yaml_file, "r") as file:
-            question_content = yaml.load(file)
-            return question_content
-
     def _populate_section(self, section):
         section["questions"] = [
```

Comment:
Modifies the shared `section` dictionary from yaml loader. Should probably be a method of `YAMLLoader`.